tests/csapi/rooms_members_local_test.go

```diff
@@ -23,7 +23,7 @@ func TestMembersLocal(t *testing.T) {
 		}),
 	)
 
-	_, incrementalSyncToken := alice.MustSync(t, client.SyncReq{})
+	_, incrementalSyncToken := alice.MustSync(t, client.SyncReq{TimeoutMillis: "0"})
```

Comment:
The spec says default `timeout` is `0`. Why do we specify it here explicitly?

Spec: https://spec.matrix.org/v1.6/client-server-api/#get_matrixclientv3sync

Reply:
I'm being pedantic, but I also vaguely remember somewhere, somewhen, an implementation using 10/30 seconds as a default like this. It makes sense, else a query-parameter-less sync request would tight-loop.

Here I'm being explicit for the sake of clarity (and pedant), is that alright?

Reply:
Yes that's fine. Please prefer being explicit.

Reply:
(just clarifying) The only reason is so that the test runs fast regardless of implementation?